item-spec/common-metadata.md

```diff
@@ -221,15 +221,15 @@ with the links section, to describe things like the Catalog an Item is in, relat
-| type       | string | [Media type](../catalog-spec/catalog-spec.md#media-types) of the referenced entity.                                                                                            |
-| title      | string | A human readable title to be used in rendered displays of the link.                                                                                                            |
-| method     | string | The HTTP method required to be supported by the target resource. `GET` by default                                                                                              |
-| headers    | object | The HTTP headers required to be sent by the target resource.                                                                                                                   |
-| body       | object | The HTTP body required to be sent by the target resource.                                                                                                                      |
+| Field Name | Type           | Description                                                                                                                                                                    |
+| ---------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| href       | string         | **REQUIRED.** The actual link in the format of an URL. Relative and absolute links are both allowed. [Trailing slashes are significant.](../best-practices.md#consistent-uris) |
+| rel        | string         | **REQUIRED.** Relationship between the current document and the linked document. See chapter "Relation types" for more information.                                            |
+| type       | string         | [Media type](../catalog-spec/catalog-spec.md#media-types) of the referenced entity.                                                                                            |
+| title      | string         | A human readable title to be used in rendered displays of the link.                                                                                                            |
+| method     | string         | The HTTP method required to be supported by the target resource. `GET` by default                                                                                              |
+| headers    | Map<string, *> | The HTTP headers required to be sent by the target resource.                                                                                                                   |
+| body       | object         | The HTTP body required to be sent by the target resource.                                                                                                                      |
```

Comment:
```suggestion
| body       | Map<string, *> | The HTTP body required to be sent by the target resource.                                                                                                                      |
```

Reply:
why the body as Map? body can be anything like the [stac api spec](https://github.com/radiantearth/stac-api-spec/blob/604ade6158de15b8ab068320ca41e25e2bf0e116/core/commons.yaml#L421), no?

Reply:
Right, you are correct. But then it's also not an object. So it should be:

```suggestion
| body       | any            | The HTTP body required to be sent by the target resource.                                                                                                                      |
```